jvm-packages/dev/package.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 by Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cd jvm-packages
+
+while true; do
+  read -p  "Do you want to skip unit tests?" yn
```

Comment:
@CodingCat, should this one be replaced with allowing the user to pass `-DskipTests`?

E.g. `./dev/build-linux.sh -DskipTests`

We should only need this if you plan to run this without user present.